code-of-conduct.md

```diff
@@ -1,23 +1,27 @@
 
-1. **Honor Boundaries**: Consent is key in roleplay. Avoid forcing unwelcome scenarios on others. Public events require participation by presence, but remember to seek approval for any significant environmental impact. Keep sensitive roleplay private and use discretion in public spaces to ensure comfort for all.
+1. **Communication is Key**: Open and respectful communication, primarily through local chat, is encouraged to maintain inclusivity. While private messages have their place, consent and privacy must always be respected to prevent harassment.
 
-1. **Minimize Drama**: Approach conflicts constructively and respect the community by avoiding slander and toxicity. Strive for peaceful resolutions and maintain a positive environment.
+1. **Consent and Boundaries**: Consent is the cornerstone of interaction here. Avoid imposing scenarios on others without their agreement. Public roleplay should not infringe on community comfort, and sensitive content must be approached with discretion. We strictly prohibit excessive violence and discriminatory language, both in public and private settings.
 
-1. **Enhance, Don’t Replace**: Use shouts, dances, and gestures to complement, not overshadow, roleplay. Excessive or disruptive gestures, along with out-of-character shouts, are not permitted. Griefing, trolling, and unsolicited promotions are grounds for disciplinary action.
+1. **Drama and Disruptions**: Constructive conflict resolution is essential. Slander, toxicity, and behaviors that disturb the peace are not tolerated. Scheduled events should not be interrupted without prior consent, emphasizing respect for the planning and effort involved.
 
-1. **Respect Events**: Do not interrupt scheduled events without prior agreement with the organizers. Understand the effort behind event planning and maintain a collaborative spirit.
+1. **Roleplay Etiquette**: Enhancements like shouts and gestures should support, not overshadow, roleplay. We discourage any form of griefing, trolling, and spamming, as these actions detract from our collective experience.
 
-1. **Respect Privacy**: Private spaces are off-limits without explicit permission. Circumventing locks or invading personal areas without consent is prohibited.
+1. **Privacy and Personal Space**: The sanctity of private spaces is to be maintained at all times. Unauthorized entry or the circumvention of security measures is forbidden.
 
-1. **Identify Yourself**: Wear the appropriate Woodcrest group tag. Use AFK zones or tags for extended absences, and remove out-of-character tags upon returning to roleplay.
+1. **Community Identity**: Wear your Woodcrest group tag proudly. If you need to step away, please use designated AFK zones or tags to inform others.
```

Comment:
```suggestion
1. **Community Identity**: Wear your Woodcrest group tag while on sim. If you need to step away, please use designated AFK zones or tags to inform others.
```

Reply:
Accounted for in 083ca5ae6200179ab46fc45830f967d3d37ce86e